hibernate-core/src/main/java/org/hibernate/dialect/Oracle8iDialect.java

```diff
@@ -490,7 +490,7 @@ public boolean forUpdateOfColumns() {
 
 	@Override
 	public String getQuerySequencesString() {
-		return "select * from user_sequences";
+		return "select * from all_sequences";
```

Comment:
I am the original committer to change `all_sequences` to `user_sequences`. Guess I made a mistake.